### PR TITLE
use xonsh for subprocess calls (pending xonsh PR and release)

### PR DESCRIPTION
**Pull request by CJ-Wright**
This should help with windows issues (since I think xonsh does the correct thing on windows where subprocess may not)

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@scopatz I think this is ready for review!

**Comment by CJ-Wright** [comment]
Now with xonsh released this is truly ready for review @sbillinge @scopatz

File: tests/outputs/preslist/presentations.aux [hunk 1]
```diff
@@ -0,0 +1,6 @@
+\relax
```
Comment (sbillinge) [inline tests/outputs/preslist/presentations.aux]: why do we have a  latex aux file in the repo?  Isn't this a file created during latex compilation?

File: tests/outputs/preslist/presentationsNotes.bib [hunk 2]
```diff
@@ -0,0 +1,2 @@
+@CONTROL{REVTEX41Control}
```
Comment (sbillinge) [inline tests/outputs/preslist/presentationsNotes.bib]: This is also generated during compilation of the latex

**Comment by scopatz** [comment]
Thanks @CJ-Wright!

**Pull request merged**